### PR TITLE
Changing ETL summary report

### PR DESCRIPTION
**Pull request by simonarocco**
## Briefly describe the feature or bug that this PR addresses.
- added test adjustment stats to the ETL summary report
- put the before and after table counts into a table

## Please link the Airtable ticket associated with this PR.
- https://airtable.com/tbli2lWQHAqBa6ZcI/viwp6zUTCqlERLzMj/recYhjPCw9l8axi5Y?blocks=hide
- https://airtable.com/tbli2lWQHAqBa6ZcI/viwp6zUTCqlERLzMj/rec60LLkkJRsD3lI6?blocks=hide

## Describe the steps you took to test the feature/bugfix introduced by this PR.
- ran ETL successfully and not successfully and checked output in dev logging etl

### CODE REVIEW

File: app/utils/notifications_sender.py [hunk 1]
```diff
@@ -47,3 +47,14 @@ def send_slack_message(message, channel='#dev-logging-backend'):
     client = WebClient(bot_token)
     client.chat_postMessage(channel=channel, text=message)
     return
+
+
+def upload_slack_file(filename, caption="Here's my file :smile:", channel='#dev-logging-etl'):
```
Comment (atatmaja) [inline app/utils/notifications_sender.py:52]: lol love the default cap

File: app/database_etl/summary_report_generator/summary_report.py [hunk 2]
```diff
@@ -40,24 +50,58 @@ def get_human_readable_time(self, timestamp: float):
...
-            body += f"Table counts after: " + f"```{json.dumps(self.table_counts_after, indent=4)}```\n"
             body += f"Status: SUCCESS"
+            self.get_table_counts_diff(self.table_counts_before, self.table_counts_after)
+            upload_slack_file(filename='etl_summary_report.png', caption=body)
+            os.remove('etl_summary_report.png')
         else:
-            body += f"Status: FAIL"
-        send_slack_message(body, channel="#dev-logging-etl")
+            body += f"Status: FAIL\n"
+            body += f"Table counts before: " + f"```{json.dumps(self.table_counts_before, indent=4)}```\n"
```
Comment (atatmaja) [inline app/database_etl/summary_report_generator/summary_report.py:99]: worth it to put this into a table as well or nah?

Comment (simonarocco) [reply app/database_etl/summary_report_generator/summary_report.py:99]: ya i can put it into a one column table just for consistency

File: app/database_etl/summary_report_generator/summary_report.py [hunk 3]
```diff
@@ -40,24 +50,58 @@ def get_human_readable_time(self, timestamp: float):
...
     def send_summary_report(self):
         body = f"Summary report for ETL run at {self.get_human_readable_time(self.start_time)}:\n\n"
-        body += f"Total runtime: {self.get_elapsed_time()} seconds\n"
+        body += f"Total runtime: {self.get_elapsed_time()} minutes\n"
         if hasattr(self, 'num_airtable_records'):
             body += f"Number of airtable records queried: {self.num_airtable_records}\n"
-        body += f"Table counts before: " + f"```{json.dumps(self.table_counts_before, indent=4)}```\n"
+        if hasattr(self, 'num_test_adjusted_records'):
+            body += f"Number of estimates test adjusted: {self.num_test_adjusted_records}\n"
+        if hasattr(self, 'num_divergent_estimates'):
```
Comment (atatmaja) [inline app/database_etl/summary_report_generator/summary_report.py:90]: let's also add how long test adjustment took here

**Review by atatmaja — APPROVED** [review]
LGTM!

**Pull request merged**